src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo,
+  user: User[],
```

Comment:
```suggestion
  users: User[],
```
it seems it is not a user, but a list of users, name it properly